docs/core/extensions/audit-reports.md

```diff
@@ -0,0 +1,327 @@
+---
+title: Audit reports for privacy and compliance
+description: Learn how to generate audit reports for privacy-sensitive data and metrics using Microsoft.Extensions.AuditReports in .NET.
+author: IEvangelist
+ms.author: dapine
+ms.date: 10/20/2025
+ms.topic: concept-article
+ai-usage: ai-assisted
+---
+
+# Audit reports for privacy and compliance
+
+The [`Microsoft.Extensions.AuditReports`](https://www.nuget.org/packages/Microsoft.Extensions.AuditReports) NuGet package provides functionality to generate audit reports about the code being compiled. These reports are particularly useful for privacy audits, compliance reviews, and understanding what telemetry data your application collects and transmits.
+
+## Why use audit reports
+
+Audit reports help organizations maintain compliance and transparency:
+
+- **Privacy compliance**: Identify all places where privacy-sensitive data is accessed or logged.
+- **Telemetry tracking**: Understand what metrics and telemetry your application generates.
+- **Code review**: Review data classification usage across your codebase.
+- **Compliance audits**: Provide documentation for compliance and security audits.
+- **Data governance**: Ensure data handling practices align with organizational policies.
+
+## Get started
+
+The `Microsoft.Extensions.AuditReports` package is a build-time tool that generates reports during compilation. Install it as a development dependency:
+
+### [.NET CLI](#tab/dotnet-cli)
+
+```dotnetcli
+dotnet add package Microsoft.Extensions.AuditReports
+```
+
+### [PackageReference](#tab/package-reference)
+
+```xml
+<PackageReference Include="Microsoft.Extensions.AuditReports" Version="9.9.0" />
+```
+
+---
+
+For more information, see [dotnet add package](../tools/dotnet-add-package.md) or [Manage package dependencies in .NET applications](../tools/dependencies.md).
+
+## Report types
+
+The package can generate three types of reports:
+
+### Metrics report
+
+Reports on the usage of source-generated metric definitions in your code. This helps you understand what metrics your application produces.
```

Comment:
Link to the documentation of source-generated metrics so that users can see which API they have to use in order to let Microsoft.Extensions.AuditReports understand what their program does.